packages/volto/src/components/theme/Search/Search.jsx

```diff
@@ -109,7 +109,11 @@ class Search extends Component {
     const options = qs.parse(this.props.history.location.search);
     this.setState({ currentPage: 1 });
     options['use_site_search_settings'] = 1;
-    this.props.searchContent('', options);
+    const { settings } = config;
+    this.props.searchContent('', {
+      ...options,
+      b_size: settings.defaultPageSize,
```

Comment:
Switch the order of lines here, otherwise it's impossible to set a different page size

Reply:
Nice catch. Thanks!